crates/apollo-mcp-server/src/graphql.rs

```diff
@@ -74,11 +79,21 @@ pub trait Executable {
             }
         }
 
-        reqwest::Client::new()
+        let client = ClientBuilder::new(reqwest::Client::new())
+            .with_init(Extension(OtelName("graphql-client".into())))
```

Comment:
Should we include 'mcp' somewhere in the name for clarity, or would it already be distinguishable enough based on other attributes?